roboteam_ai/src/skills/Keeper.cpp

```diff
@@ -27,13 +27,45 @@ void Keeper::onInitialize() {
 
 Keeper::Status Keeper::onUpdate() {
         Vector2 ballPos = World::getBall()->pos;
-        Vector2 blockPoint = computeBlockPoint(ballPos);
+        Vector2 blockPoint;
```

Comment:
idea is cool but magic numbers and this might not be the place. We could also integrate this with defenders

Reply:
Done